app/controllers/photos_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PhotosController < ApplicationController
+  def index
+    @photos = Photo.all
```

Comment:
`all` is going to pull a *lot* of items out of the database at scale.

Look at using `will_paginate` gem to add pagination very very easily.